lib/puppet_x/puppetlabs/registry/provider_base.rb

```diff
@@ -128,6 +128,8 @@ def hkeys
         {
           :hkcr => Win32::Registry::HKEY_CLASSES_ROOT,
           :hklm => Win32::Registry::HKEY_LOCAL_MACHINE,
+          :hku  => Win32::Registry::HKEY_USERS,
+          :hkcu => Win32::Registry::HKEY_USERS,
```

Comment:
Bah, busted.  I left that by accident when i was tinkering with the idea of a "hcku" root, but ended up just doing "hku" + a custom fact to get the current user, or supply your own.  I'll fix